source/from-fixture.ts

```diff
@@ -9,33 +9,42 @@ export function fromFixture<TMessageIds extends string>(
   fixture: string,
   invalidTestCase?: {
     output?: string;
-    suggestions?: eslint.SuggestionOutput<TMessageIds>[] | null | undefined;
+    suggestions?:
+      | readonly eslint.SuggestionOutput<TMessageIds>[]
+      | null
+      | undefined;
```

Comment:
When https://github.com/typescript-eslint/typescript-eslint/pull/3844 gets in, it could be simply:

```suggestion
    suggestions: eslint.TestCaseError<TMessageIds>["suggestions"];
```